web/js/components/sidebar/paletteLegend.js

```diff
@@ -31,6 +33,11 @@ class PaletteLegend extends React.Component {
 
   componentDidMount() {
     this.updateCanvas();
+    this.setState(() => {
+      return {
+        scrollContainerEl: document.querySelector('#productsHolder .simplebar-wrapper')
```

Comment:
I think this needs to be relative to the active compare side to get it to work when the B state is active.

Reply:
Fixed 👍

Reply:
I needed to change the selector to just `#productsHolder` to make it work - with `.simplebar-wrapper` included I would still see the tooltip on hover even with sidebar collapsed.

Reply:
It sounds like you might be seeing the issue that was fixed in #2480?